Client/Frontend/Login Management/AddCredentialViewController.swift

```diff
@@ -12,7 +12,7 @@ enum AddCredentialField: Int {
     case passwordItem
     
     var indexPath: IndexPath {
-        return IndexPath(row: rawValue, section: 0)
+        IndexPath(row: rawValue, section: 0)
```

Comment:
I won't comment on all the returns on the PRs, but we normally keep the returns in this codebase (makes it more explicit). For the sake of consistency please add the returns back please 🙏

Reply:
No problem. I use AppCode for development and have some settings that would highlight nonessential returns, so my eyes are drawn to it. Same goes for uses of `self`.